db/migrate/20170329064703_create_user_emails.rb

```diff
@@ -1,16 +1,20 @@
 class CreateUserEmails < ActiveRecord::Migration
   def up
     create_table :user_emails do |t|
-      t.references :user
+      t.integer :user_id
```

Comment:
@SamSaffron I removed the `t.references` and added a couple of other indexes. I haven't fully wrapped my head around indexes, though, so I'm not completely sure if they make sense.